config/routes.rb

```diff
@@ -32,5 +32,6 @@
   get '/favorites', to: 'favorites#index'
   patch '/favorites/:pet_id', to: 'favorites#update'
   delete '/favorites/:pet_id', to: 'favorites#destroy'
+  delete '/favorites', to: 'favorites#destroy'
```

Comment:
I'm a little unsure about this part

Reply:
This makes sense to me... I can't think of another RESTful way to do it.